spring-kafka-docs/src/main/asciidoc/appendix.adoc

```diff
@@ -56,37 +41,35 @@ When using 2.8.1, you also need to exclude `spring-kafka-test` 's `zookeeper` de
 
 <dependency>
     <groupId>org.apache.kafka</groupId>
-    <artifactId>kafka_2.13</artifactId>
```

Comment:
Why is this removed?

Reply:
Same answer as for same dependency in Gradle config below.

Reply:
Confused. Why do we still need `kafka-streams`? Boot manages that too.

Reply:
spring-kafka too

Reply:
Ignore me 😄

Reply:
Correct, it does manage them, but doesn't not provide.
We still have to specify artifacts in our poms, even if without versions.
The `<classifier>test</classifier>` are not managed by Boot, so we have to specify them explicitly since just plain version does not have an effect.